command/helpers.go

```diff
@@ -75,6 +75,116 @@ func formatTimeDifference(first, second time.Time, d time.Duration) string {
 	return second.Truncate(d).Sub(first.Truncate(d)).String()
 }
 
+// fmtInt formats v into the tail of buf.
+// It returns the index where the output begins.
+func fmtInt(buf []byte, v uint64) int {
+	w := len(buf)
+	for v > 0 {
+		w--
+		buf[w] = byte(v%10) + '0'
+		v /= 10
+	}
+	return w
+}
+
+// prettyTimeDiff prints a human readable time difference.
+// It uses abbreviated forms for each period - s for seconds, m for minutes, h for hours,
+// d for days, mo for months, and y for years. Time difference is rounded to the nearest second,
+// and the top two least granular periods are returned. For example, if the time difference
+// is 10 months, 12 days, 3 hours and 2 seconds, the string "10mo12d" is returned. Zero values return the empty string
+func prettyTimeDiff(first, second time.Time) string {
+	// handle zero values
+	if first.Second() == 0 {
+		return ""
+	}
+	// round to the nearest second
+	first = first.Round(time.Second)
+	second = second.Round(time.Second)
+
+	// calculate time difference in seconds
+	d := second.Sub(first)
+	u := uint64(d.Seconds())
+
+	var buf [32]byte
```

Comment:
I used a buffer for this similar to Duration.String() to prevent unnecessary allocations as we build up the output string.